doc/user/content/concepts/views.md

```diff
@@ -15,104 +15,119 @@ aliases:
 Views represent queries that are saved under a name for reference. Views provide
 a shorthand for the underlying query.
 
-Materialize offers the following types of views:
-
 Type |
 -----|------------
-[**Non-materialized views**](#non-materialized-views) | Results are **not** persisted in durable storage. Non-materialized views can be indexed to maintain and incrementally update results in memory.
+[**Views**](#views) | Results are **not** persisted in durable storage. These views can be indexed to maintain and incrementally update results in memory.
 [**Materialized views**](#materialized-views) | Results **are** persisted and incrementally updated in durable storage. Materialized views can be indexed to maintain the results in memory.
 
 All views in Materialize are built by reading data from
 [sources](/concepts/sources) and other views.
 
-## Non-materialized views
-
-A non-materialized view, also generally referred to as a "view", saves a query
-under a name to provide a shorthand for referencing the query. The query is
-**not** executed during non-materialized view creation, and a non-materialized
-view does **not** persist its results in durable storage.
+## Views
 
-**However**, you can [index](/concepts/indexes/) a non-materialized view to
-**maintain and incrementally update view results** in memory within a cluster.
-This enables queries within that [cluster](/concepts/clusters/) to use the index
-to access view results from memory.  See [Indexes and non-materialized
-views](#indexes-and-non-materialized-views) for more information.
+A view saves a query under a name to provide a shorthand for referencing the
+query. Views can be referenced across [clusters](/concepts/clusters/).
 
-See also:
+During view creation, the underlying query is not executed , and the view
+results are not persisted in durable storage.
 
-- [`CREATE VIEW`](/sql/create-view)
+```mzsql
+CREATE VIEW my_view_name AS
+  SELECT ... FROM ...  ;
+```
 
-### Indexes and non-materialized views
+<red>However</red>, in Materialize, you can [index](/concepts/indexes/) a view
+to **maintain and incrementally update** view results in memory within a
+cluster.
 
-Indexes can improve query performance. In Materialize, non-materialized views
-can be [indexed](/concepts/indexes/). Indexing a non-materialized view causes
-view results to be **maintained and incrementally updated in memory** within
-that [cluster](/concepts/clusters/). The in-memory up-to-date results are
-accessible to queries within the cluster, even for queries that do not use the
-index key(s).
+```mzsql
+CREATE INDEX idx_on_my_view ON my_view_name(...) ;
+```
 
-Indexes are local to a cluster.
+Queries within that cluster can use the index to access view results from
```

Comment:
We keep mentioning "a cluster", "the cluster", "that cluster", but never explicitly associate anything with a cluster.

```sql
CREATE INDEX idx_on_my_view IN CLUSTER active_cluster ON my_view (...);
```

Reply:
Ah .. I was thinking people might be issuing this in the console, and hence, skipping the cluster part.  Although ... I think having this content in the clusters page is nice ... and then linking from this page to that section in the clusters page.

Reply:
Think it's still nice to be verbose in this case to get the idea across!

Reply:
Sounds good.  And, you are correct ... definitely repetition helps as we don't know what page/section our users might land.